### ISSUE TITLE
Scoreboard issues

### ISSUE THREAD

**Issue by cyberpwnn** [VolmitSoftware/Gloss#26]
When trying to create a header and footer RGB (&q) bar, the bottom one seems to only show up, while the top one doesn't (space in-between does though.) There's also a flicker every now and then, might be due to a glitchy thing of both bars. I added a space between them, and tried it with characters inside too, didn't make any difference except the characters show up on line 2 but the bar still doesn't show up above. I've also tried adding &r at the end of both, thought it might be an issue with that, didn't make a difference.

```json
{
    "permission": "default",
    "title": "&s&3&lGLOSS BOARD",
    "content": [
        "&q----------------------",
        " ",
        "&q----------------------"
    ],
    "primary": true
}
```

After some more testing, the issue with the header bar not showing up is it seems that two RGB bars won't work together, while a shiny header and a RGB footer will work together, but flicking still continues.

```json
{
    "permission": "default",
    "title": "&s&3&lGLOSS BOARD",
    "content": [
        "&s&8----------------------",
        " ",
        "&q----------------------"
    ],
    "primary": true
}
```

**Comment by cyberpwnn** [comment]
Flickering is only possible if you exceed 16 characters. Gloss attempts to compress text formatting. I.e. removing formatting that wont change the text.

**Comment by cyberpwnn** [comment]
Split from #20

**Comment by cyberpwnn** [comment]
Fixed in 1.0.14
